pkgs/applications/editors/jupyter/console.nix

```
@@ -0,0 +1,37 @@
+{ python3
+, jupyter-kernel
+, lib
+}:
+
+let
+  mkConsole = {
+    definitions ? jupyter-kernel.default
+    , kernel ? null
+  }:
+    (python3.buildEnv.override {
+      extraLibs = [ python3.pkgs.jupyter_console ];
```

```suggestion
      extraLibs = [ python3.pkgs.jupyter-console ];
```
Sorry, I renamed it recently.